Online/Resource/OnlineResource.Entities.cs

```diff
@@ -8,6 +8,7 @@ public abstract partial class OnlineResource
         public Dictionary<OnlineEntity.EntityId, OnlineEntity.EntityDefinition> registeredEntities;
         public Dictionary<OnlineEntity.EntityId, EntityMembership> joinedEntities;
         public List<OnlineEntity> activeEntities;
+        public List<OnlineEntity> pendingEntities = new();  // tried to EnterResource before active
```

Comment:
Just wanna ping @henpemaz regarding the pending entities addition so we're all on the same page with the queuing

Reply:
no longer using pendingEntities